web-frontend/src/pages/TransferCoins/SendCoins/Send.tsx

```diff
@@ -0,0 +1,170 @@
+import { useResource } from '../../../custom/api/useResource';
+import { AccountObject } from '../../../models/AccountModel';
+import Intl from '../../../i18n/i18n';
+import { useLocation } from 'react-router-dom';
+import { parseForm } from '../../../utils/parseForm';
+import { useState, ChangeEvent, FormEvent } from 'react';
+import {
+  formatStandard,
+  reverseFormatStandard,
+  checkRecipientFormat,
+  Unit,
+} from '../../../utils/MassaFormating';
+import { SendForm } from './SendForm';
+import { SendConfirmation } from './SendConfirmation';
+
+interface IErrorObject {
+  amount?: string;
+  fees?: string;
+  recipient?: string;
+  error?: string;
```

Comment:
I am having a hard time to understand that interface.

An error should be generic as much as possible. Is there any reason to have fees, receipt and amount in a error object!? 🤔